docs/components/base/_index.md

```diff
@@ -17,11 +17,17 @@ hide_children: true
 # SMEs: Steve B
 ---
 
-A base is the platform that the other parts of a mobile robot attach to.
+Bases provide an API for moving all configured components attached to a platform as a whole without needing to send commands to individual components.
 
-By configuring a base component, organizing individual components to produce coordinated movement, you gain an interface to control the movement of the whole physical base of the robot without needing to send separate commands to individual motors.
+If you have a platform for a robot with other components, such as wheels or legs, attached to it, use a _base_ component.
```

Comment:
usage of the term components to refer to legs is a little confusing here

Reply:
Maybe this?
```suggestion
If you have a mobile robot, configure a base component to coordinate the motion of its motor components.
```

Reply:
```suggestion
If you have a mobile robot, use a _base_ component to coordinate the motion of its motor components.
```